packages/node_modules/@webex/plugin-meetings/src/meetings/util.js

```diff
@@ -27,6 +29,7 @@ MeetingsUtil.extractDestination = (destination, type) => {
 
 // ** This is a workaround to fix Firefox and google android not downloading H264 codec on start
 MeetingsUtil.triggerH264Codec = async () => {
```

Comment:
Is this function needed any more? Can we replace the call to it with a call to `checkMediaPlayback`?

Reply:
Probably not, just changed it. Thanks!